tests/cases/fourslash/importTypeModuleCompletions.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="fourslash.ts" />
+
+// @moduleResolution: node
+
+// @Filename: /ns.ts
+////export namespace Foo {
+////    export namespace Bar {
+////        export class Baz {}
+////    }
+////}
+// @Filename: /node_modules/package/index.ts
+////export namespace Foo {

```

Comment:
Path completions shouldn't even need to read the file, so this could just be `//// file content not read`